roles/observe_chargeback/tasks/chargeback_tests.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Enable Cloudkitty Module (hashmap)
+  ansible.builtin.command:
+    cmd: "{{ openstack_cmd }} rating module enable hashmap"
+  register: enable_hashmap
+  # The task is only 'changed' if the output confirms the module was enabled/changed
+  changed_when: True
+  failed_when: enable_hashmap.rc != 0
+
+- name: Enable Cloudkitty Module (noop)
+  ansible.builtin.command:
+    cmd: "{{ openstack_cmd }} rating module enable noop"
+  register: enable_noop
+  changed_when: True
+  failed_when: enable_noop.rc != 0
```

Comment:
This is enabled by default, and we don't need to enable it again because it's not used